sod-server/src/commands/base/PlaceRoadCommand.ts

```diff
@@ -14,3 +15,25 @@ export class PlaceRoadCommand extends Command<MyRoom, Payload> {
     }
   }
 }
+
+export class PlaceRoadWithPayCommand extends Command<MyRoom, Payload> {
+  execute(payload: Payload) {
+    const player = this.room.state.players.get(payload.playerId)
+    const state = this.room.state
+
+    let cardsToBePayedToBank = []
+    cardsToBePayedToBank.push(...player.cardsOfTypeWithAmount(state, CardNames.Brick, 1))
```

Comment:
Ístaðin fyri at brúka cardsOfTypeWithAmount, so kanst tú gera:
```typescript
...player.cardsOfType(state, CardNames.Brick).slice(0, 1)
```
`slice()`, tekur `x` elementir frá positión y, sum omanfyri er x = 1, y = 0.
So sleppur tú undan at hava eina so 'kompleksa' funktión sum `cardsOfTypeWithAmount`.

Hetta ger eisini at tað ikke er neyðugt at tjekka: `cardsToBePayedToBank.includes(null)`